### ISSUE TITLE
From lunar : change name to match python naming

### ISSUE THREAD

**Issue by asmodehn** [asmodehn/pyros-common#1]
This is because pyros structure has now changed to rely on namespace packages (from setuptools)

**Comment by asmodehn** [comment]
this is not as simple as I thought before.

Normal packages are supported in python 2 and 3 and ROS deb packages
Namespace packages are supported in python 3 only (properly with PEP 420 which works with ROS as well) and ROS with python3.

We can probably rename our package to "pyros_interfaces_common" from kinetic, to have all interfaces packages with a proper prefix in package.
But maintaining a namespace package will be doable only when ROS will support python3 officially (still some way to go AFAIK...), so very likely in another distro, some time away in the future.

**Comment by asmodehn** [comment]
Or maybe we keep pyros_common as PyPI and pip name for now... even if we have 2 python packages named : 
- pyros_interfaces_common
- pyros_interfaces_mock

And we will change package name only once, when we move to python3 and can support python namespace packages in deb pkgs.

**Comment by asmodehn** [comment]
Everything seems to work properly with our current packaging naming : 
```
$ pip list | grep pyros
pyros-common (0.4.2)
pyros-config (0.2.0)
pyros-interfaces-ros (0.4.0)
pyros-setup (0.2.1)
pyros-test (0.0.6)
pyros-utils (0.1.3)
$ python -c "import pyros_interfaces_common; print pyros_interfaces_common.__file__"
WARNING:root:ZMQ : Protobuf message implementation not found. Using pickle based protocol
/opt/ros/indigo/lib/python2.7/dist-packages/pyros_interfaces_common/__init__.pyc
```

**Comment by asmodehn** [comment]
We now have a `pyros_common` package ,among the 3 others.

Renaming is on hold until we stabilize the structure of pyros, between "common" parts (code/data that will be serialized and passed around) and "interface" parts (code/data that stays in one process to interface).

The interface parts are currently splitted in two because we have a base class, but there may be better ways to do interfaces/facades in python...
